Sources/DarwinPlatform/Darwin.swift

```diff
@@ -0,0 +1,144 @@
+import Foundation
+import SwiftlyCore
+@_implementationOnly import Archive
```

Comment:
Why is this @_implementationOnly? I'd rather we didn't include `_` directives if they can be avoided.

Reply:
I think Archive(Or CLibArchive) is an internal detail of the implementation. So I use `@_implementationOnly import` to imply the fact and also prevent use from using CLibArchive symbol in any public API of the module in compile time.

`@_implementation import` and `@_exported import` is widely used in Swift ecosystem and there is some discussion in the forum to make them stable - remove "_".

But since swiftly currently only provide an executable produce and would not be another package's dependency. It's OK to use the normal import.